src/main/scala/mesosphere/marathon/MarathonScheduler.scala

```diff
@@ -71,7 +73,7 @@ class MarathonScheduler @Inject() (
   override def statusUpdate(driver: SchedulerDriver, status: TaskStatus): Unit = {
     log.info("Received status update for task %s: %s (%s)"
       .format(status.getTaskId.getValue, status.getState, status.getMessage))
-
+    healthCheckManager.reconcile(status)
```

Comment:
Not really with the current implementation: If a statusUpdate received during a reconciliation does not change the state of a task, no StateChange is populated, and we can't use the update step processor.